lambda/matching.ml

```diff
@@ -3223,14 +3227,14 @@ and do_compile_matching ~scopes repr partial ctx pmh =
       let ploc = Pattern_head.loc ph in
       match Pattern_head.desc ph with
       | Any ->
-         compile_no_test ~scopes divide_var Context.rshift repr partial ctx pm
-      | Tuple l ->
-          compile_no_test ~scopes (divide_tuple ~scopes l pomega)
+          compile_no_test ~scopes divide_var Context.rshift repr partial ctx pm
```

Comment:
Would you mind removing the white space change here?

Reply:
2986beaa7800deef612ffb1b77a9b0a62a9b2318 introduced this inconsistent indentation (a single space when all other cases have two), I believe that fixing it is a reasonable move here.